index.html

```diff
@@ -376,15 +510,40 @@ <h3 class="recent-work-article-header">Multi-post-story</h3>
           </article>
         </div>
 
-        <div class="recent-work-div">
+        <div class="recent-work-div display-flex flex-d-col flex-d-row">
+          <article class="recent-work-article display-none-sm display-block-lg">
+            <h3 class="recent-work-article-header">Multi-post-story</h3>
+            <p class="recent-work-paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul
+              class="recent-work-ul display-flex justify-content-space-between"
+            >
+              <li class="recent-work-ul-li">css</li>
+              <li class="recent-work-ul-li">html</li>
+              <li class="recent-work-ul-li">boostrap</li>
+              <li class="recent-work-ul-li border-none">Ruby</li>
+            </ul>
+            <button
+              type="button"
+              aria-label="see project"
+              class="see-project-button border-none"
+            >
+              see projects
```

Comment:
- [ ] Please, consider grammatical issues on the page like this, line 535 - fix it to "See project" 👍